frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/footer.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { ActionGroup, Button } from '@patternfly/react-core';
+import { ModalComponentProps } from '@console/internal/components/factory/modal';
+
+import { POOL_PROGRESS } from '../../../constants/storage-pool-const';
+import {
+  BlockPoolState,
+  BlockPoolAction,
+  BlockPoolActionType,
+  checkRequiredValues,
+} from '../../../utils/block-pool';
+
+export const StoragePoolModalFooter = (props: StoragePoolFooterComponentProps) => {
```

Comment:
can we generalize this and use one footer component at both places ?

Reply:
I tried, it will lead to a very high coupled code, and unnecessary confusion.

Reply:
Even small change required anyone of the page in the future then lot of hacks and flags will be required, so better we maintain different footer, for update i will re-use one of these footer

Reply:
okay